### DISCUSSION TITLE
PWA feature not working.

### DISCUSSION THREAD

**Discussion by abhishek7h** [cotes2020/jekyll-theme-chirpy#1320] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I have the PWA (progressive web apps) feature toggled on. It used to work perfectly for the first couple of days, but now, it doesn't show up when I look up my website. I have tried this on Chrome and Samsung Internet. Previously, it used to say "Install site" but now it doesn't say that and I can't even manually install it from the three dots.

`pwa:
  enabled: true # the option for PWA feature`

### Operations you have already tried

I have tried disabling it and enabling it once again but that didn't really do anything.

### Anything else?

Nope

**Comment by rafisics** [comment]
Hi, can you let me know who to get PWA back? I am having kind of the same issue: https://github.com/cotes2020/jekyll-theme-chirpy/discussions/2258.
